### ISSUE TITLE
Is this Debloater removing bundles?

### ISSUE THREAD

**Issue by omgfrost** [Sycnex/Windows10Debloater#263]
Hello,
like the title says when i Powershell this command: `Get-AppxPackage -AllUsers -PackageTypeFilter Bundle | Select-Object -Property Name, PackageFullName` and used custom list to remove all of them (i think it's removing) instead i got this:
![UPevswq 1](https://user-images.githubusercontent.com/14861120/86333173-d62c0680-bc4b-11ea-9f75-7b30204e8f9f.png)
Used another script to remove something like:
Microsoft.549981C3F5F10
that should be the real cortana, but after removing these bundles (by removing registry keys too) my system became unstable and it froze for a while and after that i got BSOD with error code DPC_WATCHDOG_VIOLATION.
I tried all the fixes possible for that but in the end i repair\reinstall windows over with media creation tool maintaning files but not app.
Turned out i've had all programs,  settings and apps removed so i had to reinstall all but my files were there. (to reinstall i just install programs then copy relative folders in appdata\local and appdata\roaming from windows.old folder.
In conclusion i'm asking:
Is this debloater not removing these bundle for security\stability purpose or they are not still added and tested before release a new version?
Thanks in advance
Frost

**Comment by R4vz** [comment]
I would recommend you to use SysPrepDebloater in [Audit Mode](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/boot-windows-to-audit-mode-or-oobe). Before that make sure you modify the script as per your needs.

**Comment by omgfrost** [comment]
But with sysprep i need to fresh install windows and right now i dont want to lose all my stuff neither i got time to backup all my (a lot) of files\archives. So i just want to know if there's a way for this debloater to remove these bundles as i did with another script and put my system into an instability so i had to repair\reinstall over with files but not apps and this leaded me to indeed have files but reinstalling all programs and needed apps, settings them again and run debloater again in the end.
I wouldnt do this again...
